indexer-app/src/cmr/indexer/data/concepts/collection.clj

```diff
@@ -187,6 +199,8 @@
          publication-references :PublicationReferences
          collection-citations :CollectionCitations} collection
         parsed-version-id (collection-util/parse-version-id version-id)
+        direct-distribution-information (umm-c-ddi->es-doc-ddi
```

Comment:
No need to retrieve the unused values from UMM-C.

Reply:
They need to be converted someplace. The s3-bucket-and-prefix-names value doesn't come in by itself, it's part of a DirectDistrubtionInformationType.

Reply:
We don't need all the values in DirectDistrubtionInformationType, you can remove `umm-c-ddi->es-doc-ddi` function and just parse out `s3-bucket-and-object-prefix-names` here.